### ISSUE TITLE
bot not working

### ISSUE THREAD

**Issue by yildirimkursat** [Qwerty-Space/AntiDeletedAccountsBot#9]
bot is not working.  The bot hasn't been working for months when it should run every 6 hours

**Comment by Qwerty-Space** [comment]
Yeah, the bot was turned off by my hosting provider. So I need to re-work it and turn it on again. 

I've just had a lot of things in life get in the way, it's not high on my priorities at the moment.

**Comment by yildirimkursat** [comment]
I understand.  I hope the bot works soon.  good work.

**Comment by ramack19** [comment]
Any progress on getting this bot working?

**Comment by Qwerty-Space** [comment]
> Any progress on getting this bot working?

My life has been pretty busy for the past 6 months.  Any help would be appreciated.

**Comment by ramack19** [comment]
Hi QS,
Not sure how I could help, but PM me for info.

Since the post a few days ago, I did find an alternative to a Tele bot. A pretty simple Python script that I can run from the CLI.

**Comment by Qwerty-Space** [comment]
> Hi QS, Not sure how I could help, but PM me for info.
> 
> Since the post a few days ago, I did find an alternative to a Tele bot. A pretty simple Python script that I can run from the CLI.

I just need a better way to queue jobs.  It's very badly optimised currently
